Makefile

```diff
@@ -152,6 +152,10 @@ endif
 docker-push: ## Push docker image with the manager.
 	docker push ${IMG}
 
+.PHONY: clean
+clean:  ## Cleanup the build generated files
+	rm -Rf bin/ cover.out bundle_tmp*
```

Comment:
There are other make targets that will install binaries under bin/ so I think delete them via rm was the right decision (rather than `go clean`)